packages/mui-material/src/Rating/Rating.test.js

```diff
@@ -225,6 +225,12 @@ describe('<Rating />', () => {
       expect(screen.getByRole('img')).toHaveAccessibleName('Stars: 2');
     });
 
+    it('can be labelled with getLabelText when non set value', () => {
```

Comment:
This test title seems incorrect.

```suggestion
    it('should have a correct label when no value is set', () => {
```